climada/engine/test/test_impact.py

```diff
@@ -594,7 +595,7 @@ def test_local_exceedance_impact_methods(self):
 
         # test log log extrapolation
         impact_stats, _, _ = impact.local_exceedance_impact(
-            return_periods=(1000, 30, 0.1), method="extrapolate"
+            return_periods=(1000, 30, 0.1), method="extrapolate", bin_decimals=2
```

Comment:
Since it is a bit 'out of the ordinary', can you also try with `bin_decimals = -1` or any negative value?